README.md

```diff
@@ -0,0 +1,38 @@
+# OVGT - Open Source VGT Controller
+
+An open-source Variable Geometry Turbo controller for Teensy 4.1 and MicroMod Teensy platforms.
+
+## Hardware Requirements
+
+### Supported Microcontrollers
+- Teensy 4.1 (primary development target)
+- MicroMod Teensy (secondary target)
+
+### Required Components
+- 20x4 I2C LCD Display
+- CAN transceiver compatible with 3.3V logic
+
+## Pin Usage
+
+### Teensy 4.1
+| Function | Pin | Description |
+|----------|-----|-------------|
+| Actuator Control | PWM Pin 2 | PWM output at 300Hz for VGT actuator control |
+| CAN Bus | CAN2 | Communication with engine/vehicle CAN bus |
```

Comment:
This can bus is actually for communicating with the Actuator, not the engine/vehicle